src/types.ts

```diff
@@ -0,0 +1,18 @@
+import JSBI from 'jsbi'
+
+export type BigIntish = BigInt | JSBI | string | number
```

Comment:
Not sure, but I wonder if ethers has faced this problem, since they work with `bn.js` and, I think, native `BigInt`s now? Pointing this out since I can see a lot of usage between this library and the types that are available from ethers.

Reply:
They seem to accept `BigInt` now indeed: https://github.com/ethers-io/ethers.js/blob/master/packages/bignumber/src.ts/bignumber.ts#L234-L236

But their type doesn’t seem to include BigInt, it only has ether’s BigNumber, strings, numbers and ethers’ bytes. https://github.com/ethers-io/ethers.js/blob/master/packages/bignumber/src.ts/bignumber.ts#L24

In any case, ethers’ BigNumber works the same way than the other libraries: https://github.com/ethers-io/ethers.js/blob/master/packages/bignumber/src.ts/bignumber.ts#L189-L195